### PR TITLE
Added devhost:21 

### PR DESCRIPTION
**Pull request by swaaz**

### CODE REVIEW

File: src/contents/events/devHost21/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+### Can Bad Design Kill
+**Time**: 15th May, 11:00 AM - 12:00 PM
+
+In this session we will be talking about design in general and how it has grown from just being used in niche user apps to more sensitive systems that deal with medicine, digital policing, vehicular systems etc where bad UX can have more impact than just the user getting irritated. Now being a part of every such impactful industry - bad design can have unseen ramifications when it comes to effecting safety and health. We'll discuss what is design- how it has grown, followed by Bad Design and in what spectrum bad design becomes dangerous.
+#### Speaker
+- [Jehad Mohamed](https://jehadmohamedz.wixsite.com/meep/), UI/UX Designer at MobStac
+
+---
+### Getting Started with Your Journey into Cloud Security
+**Time**: Getting Started with Your Journey into Cloud Security
```
Comment (afrzhussain) [inline src/contents/events/devHost21/index.md]: title right ?

**Review by afrzhussain — CHANGES_REQUESTED** [review]

**Review by dhanushp — COMMENTED** [review]
Feedback link?

**Comment by swaaz** [comment]
image
Missed some tags. I will fix them
@coderhawk999 @dhanushp

**Comment by swaaz** [comment]
@coderhawk999 @dhanushp Fixed everything !!

**Pull request merged**